### ISSUE TITLE
Moblog Exception Caught Error on Manual Check

### ISSUE THREAD

**Issue by firebrandmedia** [ExpressionEngine/ExpressionEngine#383]
**Description of the problem**
I'm trying to use Moblog on a site and I receive an error when I use the manual check option.

**How To Reproduce**

1. Install Moblog
2. Supply email username, incoming mail server, password
3. Leave default settings as is
4. Use the "Check Now" button to manually test the settings

**Error Messages**

<pre><code>
Exception Caught
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'field_group' in 'field list':
SELECT `site_id`, `channel_title`, `channel_url`, `rss_url`, `comment_url`, `deft_comments`, `cat_group`, `field_group`, `channel_notify`, `channel_notify_emails` FROM (`exp_channels`) WHERE `channel_id` = '1'

ee/legacy/database/drivers/mysqli/mysqli_connection.php:117
Stack Trace: Please include when reporting this error

    #0 ee/legacy/database/drivers/mysqli/mysqli_driver.php(112): CI_DB_mysqli_connection->query('SELECT `site_id...')
    #1 ee/legacy/database/DB_driver.php(270): CI_DB_mysqli_driver->_execute('SELECT `site_id...')
    #2 ee/legacy/database/DB_driver.php(180): CI_DB_driver->simple_query('SELECT `site_id...')
    #3 ee/legacy/database/DB_active_rec.php(1206): CI_DB_driver->query('SELECT `site_id...')
    #4 ee/EllisLab/Addons/moblog/mod.moblog.php(936): CI_DB_active_record->get_where('channels', Array)
    #5 ee/EllisLab/Addons/moblog/mod.moblog.php(875): Moblog->post_entry()
    #6 ee/EllisLab/Addons/moblog/mcp.moblog.php(914): Moblog->check_pop_moblog()
    #7 [internal function]: Moblog_mcp->check('1')
    #8 ee/EllisLab/ExpressionEngine/Controller/Addons/Addons.php(1662): call_user_func_array(Array, Array)
    #9 ee/EllisLab/ExpressionEngine/Controller/Addons/Addons.php(946): EllisLab\ExpressionEngine\Controller\Addons\Addons->getModuleSettings('moblog', 'check', Array)
    #10 [internal function]: EllisLab\ExpressionEngine\Controller\Addons\Addons->settings('moblog', 'check', '1')
    #11 ee/EllisLab/ExpressionEngine/Core/Core.php(241): call_user_func_array(Array, Array)
    #12 ee/EllisLab/ExpressionEngine/Core/Core.php(110): EllisLab\ExpressionEngine\Core\Core->runController(Array)
    #13 ee/EllisLab/ExpressionEngine/Boot/boot.php(151): EllisLab\ExpressionEngine\Core\Core->run(Object(EllisLab\ExpressionEngine\Core\Request))
    #14 public_html/admin.php(153): require_once('...')
    #14 public_html/admin.php(153): require_once('...') 

</code></pre>

**Environment Details:**
 - Version: EE 5.3.0
 - Moblog v 3.2.0
 - PHP Version 7.0.33
 - MySQL Version 10.3.22-MariaDB-log-cll-lve
 - OS: Linux

**Possible Solution**

**Additional context**
This install of EE has 1 channel with two fields (Post - textarea, Post Media - file upload field)

I also noticed that when I check the file upload directory, a small CPanel logo is added to the directory after every "check now" instance.

**Comment by creativearc-paul** [comment]
+1. I have the same issue.

field_group is no longer a column in exp_channels, I presume.

**Comment by oftoday** [comment]
+1 I have the same problem as well.

**Comment by firebrandmedia** [comment]
I saw that this may be fixed so I updated to v5.3.2 and tested it out. I'm still seeing the same error when testing this out:

```
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'field_group' in 'field list':
SELECT `site_id`, `channel_title`, `channel_url`, `rss_url`, `comment_url`, `deft_comments`, `cat_group`, `field_group`, `channel_notify`, `channel_notify_emails` FROM (`exp_channels`) WHERE `channel_id` = '1'

ee/legacy/database/drivers/mysqli/mysqli_connection.php:117
Stack Trace: Please include when reporting this error

    #0 ee/legacy/database/drivers/mysqli/mysqli_driver.php(112): CI_DB_mysqli_connection->query('SELECT `site_id...')
    #1 ee/legacy/database/DB_driver.php(270): CI_DB_mysqli_driver->_execute('SELECT `site_id...')
    #2 ee/legacy/database/DB_driver.php(180): CI_DB_driver->simple_query('SELECT `site_id...')
    #3 ee/legacy/database/DB_active_rec.php(1206): CI_DB_driver->query('SELECT `site_id...')
    #4 ee/EllisLab/Addons/moblog/mod.moblog.php(936): CI_DB_active_record->get_where('channels', Array)
    #5 ee/EllisLab/Addons/moblog/mod.moblog.php(875): Moblog->post_entry()
    #6 ee/EllisLab/Addons/moblog/mcp.moblog.php(914): Moblog->check_pop_moblog()
    #7 [internal function]: Moblog_mcp->check('3')
    #8 ee/EllisLab/ExpressionEngine/Controller/Addons/Addons.php(1662): call_user_func_array(Array, Array)
    #9 ee/EllisLab/ExpressionEngine/Controller/Addons/Addons.php(946): EllisLab\ExpressionEngine\Controller\Addons\Addons->getModuleSettings('moblog', 'check', Array)
    #10 [internal function]: EllisLab\ExpressionEngine\Controller\Addons\Addons->settings('moblog', 'check', '3')
    #11 ee/EllisLab/ExpressionEngine/Core/Core.php(241): call_user_func_array(Array, Array)
    #12 ee/EllisLab/ExpressionEngine/Core/Core.php(110): EllisLab\ExpressionEngine\Core\Core->runController(Array)
    #13 ee/EllisLab/ExpressionEngine/Boot/boot.php(151): EllisLab\ExpressionEngine\Core\Core->run(Object(EllisLab\ExpressionEngine\Core\Request))
    #14 public_html/admin.php(153): require_once('...')
    #14 public_html/admin.php(153): require_once('...') 
```

**Comment by gfive** [comment]
problem is still not solved!!
